### PR TITLE
- Task: correctly handles all rotations, even those involving flippin…

### PR DESCRIPTION
**Pull request by Heshyo**
Correctly handles all rotations, even those involving flipping horizontally.

### CODE REVIEW

**Comment by lsmith77** [comment]
please do not use short open tags .. I assume styleci initially told you that you need to use it before you rebased .. because the default config specifies short open tags, while the config in master defines that it should not be used.

**Comment by Heshyo** [comment]
```AutoRotateFilterLoader.php``` and ```AutoRotateFilterLoaderTest.php``` both start with ```<?php```. I'm not sure where you're seeing ```<?```?

**Comment by lsmith77** [comment]
argh .. I meant short array syntax :)

File: Imagine/Filter/Loader/AutoRotateFilterLoader.php [hunk 1]
```diff
@@ -11,22 +11,27 @@
  */
 class AutoRotateFilterLoader implements LoaderInterface
 {
-    protected $orientationKeys = array(
+    protected $orientationKeys = [
```
Comment (lsmith77) [inline Imagine/Filter/Loader/AutoRotateFilterLoader.php]: please do not use short array syntax

File: Imagine/Filter/Loader/AutoRotateFilterLoader.php [hunk 2]
```diff
@@ -11,22 +11,27 @@
...
         'exif.Orientation',
         'ifd0.Orientation',
-    );
+    ];
 
     /**
      * {@inheritdoc}
      */
-    public function load(ImageInterface $image, array $options = array())
+    public function load(ImageInterface $image, array $options = [])
```
Comment (lsmith77) [inline Imagine/Filter/Loader/AutoRotateFilterLoader.php]: please do not use short array syntax

**Comment by Heshyo** [comment]
Yes I just noticed about the short array syntax. Should I do another commit or you already modified it?

**Comment by lsmith77** [comment]
yeah please fix in another commit or by force pushing on the last commit

**Comment by lsmith77** [comment]
thanks!

**Pull request merged**